File: sandbox/dotnet/src/Dropbox.SignSandbox/SignatureRequestCancelDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class SignatureRequestCancelDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        try
        {
            new SignatureRequestApi(config).SignatureRequestCancel(
                signatureRequestId: "fa5c8a0b0f492d768749333ad6fcc214c111e967"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling SignatureRequest#SignatureRequestCancel: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: sandbox/node/src/SignatureRequestCancelDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.SignatureRequestApi();

const signatureRequestId = "fa5c8a0b0f492d768749333ad6fcc214c111e967";

apiCaller.signatureRequestCancel(
    signatureRequestId,
).catch(error => {
  console.log("Exception when calling SignatureRequest#signatureRequestCancel:");
  console.log(error.body);
});
